bokeh/charts/utils.py

```diff
@@ -38,3 +44,109 @@ def polar_to_cartesian(r, start_angles, end_angles):
         points.append(cartesian(r, (end + start)/2))
 
     return zip(*points)
+
+# TODO: Experimental implementation. This should really be a shared
+#       pattern between plotting/charts and other bokeh interfaces
```

Comment:
Can you elaborate a little bit about why you need this class? I will probably have the answer below... in the CLI implementation, but it would be nice a short indication (in the code even better as a comment as with the TODO lines above).